mapoi_turtlebot3_example の package.xml に mapoi_server / mapoi_webui / mapoi_rviz_plugins の exec_depend 欠落

## 現状

`mapoi_turtlebot3_example/package.xml` には `mapoi_interfaces` のみが `<depend>` 宣言されており、launch ファイル (`mapoi_turtlebot3_example/launch/turtlebot3_navigation_launch.yaml`) で実行する以下のパッケージの依存宣言が欠落している:

- `mapoi_server` (`mapoi_server` / `mapoi_nav_server` / `mapoi_rviz2_publisher` を起動)
- `mapoi_webui` (`mapoi_webui_node.py` を起動)
- `mapoi_rviz_plugins` (RViz panel として利用、RViz 設定に含まれる)

## 影響

`colcon build --packages-up-to mapoi_turtlebot3_example` で芋づる build が効かず、上記が build されない。

この状態で `ros2 launch mapoi_turtlebot3_example turtlebot3_navigation_launch.yaml` を実行すると:

```
[ERROR] [launch]: "package 'mapoi_server' not found, searching: ['/ros2_ws/install/mapoi_turtlebot3_example', '/ros2_ws/install/mapoi_interfaces', '/opt/ros/humble']"
```

で launch が即死する。

## 対応

`mapoi_turtlebot3_example/package.xml` に以下を追加:

```xml
<exec_depend>mapoi_server</exec_depend>
<exec_depend>mapoi_webui</exec_depend>
<exec_depend>mapoi_rviz_plugins</exec_depend>
```

build-time の CMake/find_package 連鎖を避けるため `<depend>` ではなく `<exec_depend>` が適切（参考: lessons での Jazzy turtlebot3_gazebo 問題と同様の考え方）。

## 検出経緯

#32 (PR for #9) の動作確認時に発覚。

## 関連

- #9 / #32

--- a/mapoi_turtlebot3_example/package.xml
+++ b/mapoi_turtlebot3_example/package.xml
@@ -19,6 +19,9 @@
   <exec_depend>turtlebot3</exec_depend>
   <exec_depend>turtlebot3_gazebo</exec_depend>
   <exec_depend>turtlebot3_navigation2</exec_depend>
+  <exec_depend>mapoi_server</exec_depend>
+  <exec_depend>mapoi_webui</exec_depend>
+  <exec_depend>mapoi_rviz_plugins</exec_depend>
   <depend>yaml_cpp_vendor</depend>
   <depend>ament_index_cpp</depend>
   <depend>geometry_msgs</depend>

